src/components/InstancesList/InstancesList.js

```diff
@@ -491,6 +509,15 @@ class InstancesList extends React.Component {
     this.setState({ isSelectedRecordsModalOpened: false });
   }
 
+  handleImportRecordModalSubmit = (args) => {
+    this.setState({ isImportRecordModalOpened: false });
+    this.props.mutator.query.update({ _path: '/inventory/import', xid: args.externalIdentifier });
```

Comment:
@MikeTaylor I think we can use `updateLocation` here as well and remove the query resource mutator and `stripesConnect`.